src/du/du.rs

```diff
@@ -62,16 +65,39 @@ impl Stat {
             size: metadata.len(),
             blocks: metadata.blocks() as u64,
             nlink: metadata.nlink() as u64,
+            inode: metadata.ino() as u64,
             created: metadata.mtime() as u64,
             accessed: metadata.atime() as u64,
             modified: metadata.mtime() as u64,
         })
     }
 }
 
+fn get_default_blocks() -> u64 {
+    for env_var in ["DU_BLOCK_SIZE", "BLOCK_SIZE", "BLOCKSIZE"].into_iter() {
```

Comment:
It seems that `gdu` doesn't actually error if the variable value is wrong, so you could just do this instead:
```rust
for env_var in &["DU_BLOCK_SIZE", "BLOCK_SIZE", "BLOCKSIZE"] {
    if let Some(result) = env::var(env_var).ok().and_then(|val| val.parse().ok()) {
        return result;
    }
}
```

Also apparently the block size can use suffixes like `kB`.  There should be some code that can help in `uucore` or one of the other utils that deals with that sort of stuff.  If you don't want to implement that in this pull request then please open an issue so we can keep track 😄.